lib/Driver/Driver.cpp

```diff
@@ -2943,6 +2944,10 @@ Job *Driver::buildJobsForAction(Compilation &C, const JobAction *JA,
                          options::OPT_emit_objc_header_path))
     chooseObjectiveCHeaderOutputPath(C, OutputMap, workingDirectory,
                                      Output.get());
+  
+  if (C.getArgs().hasArg(options::OPT_emit_symbol_graph)) {
+    chooseSymbolGraphOutputPath(C, OutputMap, workingDirectory, Output.get());
+  }
```

Comment:
Nit: Surrounding code more-or-less conforms to the LLVM Style guide which omits braces for single-statement bodies:
[Don’t Use Braces on Simple Single-Statement Bodies of if/else/loop Statements 
](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements)